Index: pubspec.yaml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>name: marvel_rivals_mod_manager\r\ndescription: Mod Manager for Marvel Rivals 2.0\r\npublish_to: 'none'\r\nversion: 1.0.0+1\r\n\r\nenvironment:\r\n  sdk: '>=3.0.0 <4.0.0'\r\n\r\ndependencies:\r\n  flutter:\r\n    sdk: flutter\r\n  cupertino_icons: ^1.0.2\r\n  provider: ^6.1.1\r\n  path_provider: ^2.0.15\r\n  file_picker: ^6.1.1\r\n  process_run: ^0.13.1\r\n  platform: ^3.1.0\r\n  path: ^1.8.3\r\n  http: ^1.2.0\r\n  ffi: ^2.1.0\r\n  win32: ^5.0.9\r\n  shared_preferences: ^2.2.2\r\n  url_launcher: ^6.2.5\r\n  crypto: ^3.0.3\r\n  archive: ^3.4.9\r\n  flutter_localizations:\r\n    sdk: flutter\r\n  intl: ^0.19.0\r\n\r\ndev_dependencies:\r\n  flutter_test:\r\n    sdk: flutter\r\n  flutter_lints: ^2.0.0\r\n\r\nflutter:\r\n  uses-material-design: true\r\n  assets:\r\n    - assets/img/cover.png\r\n    - assets/lang/ 
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/pubspec.yaml b/pubspec.yaml
--- a/pubspec.yaml	(revision 477e1a164357c70255ef0c0b413154e81c661ea1)
+++ b/pubspec.yaml	(date 1740571981991)
@@ -12,8 +12,8 @@
   cupertino_icons: ^1.0.2
   provider: ^6.1.1
   path_provider: ^2.0.15
-  file_picker: ^6.1.1
-  process_run: ^0.13.1
+  file_picker: ^9.0.1
+  process_run: ^1.2.3
   platform: ^3.1.0
   path: ^1.8.3
   http: ^1.2.0
@@ -22,7 +22,7 @@
   shared_preferences: ^2.2.2
   url_launcher: ^6.2.5
   crypto: ^3.0.3
-  archive: ^3.4.9
+  archive: ^4.0.3
   flutter_localizations:
     sdk: flutter
   intl: ^0.19.0
@@ -30,7 +30,7 @@
 dev_dependencies:
   flutter_test:
     sdk: flutter
-  flutter_lints: ^2.0.0
+  flutter_lints: ^5.0.0
 
 flutter:
   uses-material-design: true
